scripts/database/querycount/count.pl

```diff
@@ -25,7 +25,7 @@
 
```

Comment:
For me, "diff" is zero so nothing goes into the "tail" file.

10486132 - 10486132 = 0  (`$newsize - $pglogfilesize`) 

I can work around this by setting diff to the size of the file like this:

```
 my $newsize = (stat($pglogfile))[7];
 my $diff = $newsize - $pglogfilesize;
+$diff = $pglogfilesize;
 
 system "tail -c ".$diff." < ".$pglogfile." > tail";
```

But I'm probably just confused. Not sure what's going on here.

Reply:
If the log file really isn't growing, between the start and the end of the task you are testing, maybe verbose logging isn't enabled (the `log_statement = 'all'` in the PostgresQL config). 

Wrong log file?  (it starts a new log file every day). 

What/how are you testing, anyway?

Reply:
Whoops! I simply forgot how it works and obviously didn't study the README closely enough. I was thinking that it would operate on any old postgres log file (I have hundreds of these). Now I remember (and see in the README) that I'm supposed to get the script started, then go do something, and then go back to the script and tell it I'm done. Sorry for the noise.

Reply:
Yes, that was the intended use case (human operator-defined "start" and "end" of the operation). If you want to count everything in the log file, then yes, setting that "diff" to the full size would achieve that. That's what @donsizemore is doing when using the script to count all the queries issued during as RestAssured run. 
Thinking about it, I'll add a flag for this mode - and this way Don would not need to hack the script in order to use it w/ Jenkins.